### ISSUE TITLE
Suggestion: add `npm install` to the README

### ISSUE THREAD

**Issue by yurivish** [observablehq/framework#1783]
Right now the first command in the readme is `npm run dev`, which assumes `npm install` has already been run.

**Comment by mbostock** [comment]
Are you referring to the [Getting started](https://observablehq.com/framework/getting-started) guide? The reason it doesn’t suggest `npm install` is because it asks you to run

```
npx @observablehq/framework@latest create
```

and if you follow the defaults as suggested, this command will already have run `npm install` for you. (It’s the `Install dependencies?` prompt.)

**Comment by yurivish** [comment]
Oh, sorry, I should have clarified -- I made a new Framework project today and was referring to the readme of that project. A colleague of mine who wanted to collaborate but does not regularly program in JavaScript was confused about how to get a local version of the project running when I linked them to the repo. They ran `npm run dev` but got an error since they didn't realize they first needed to run `npm install` after cloning the repository locally.

![Image](https://github.com/user-attachments/assets/b64cde26-3951-4f9e-8525-05e8d453352b)
